.github/workflows/lint.yml

```
@@ -38,3 +38,10 @@ jobs:
         run: yarn lint:cspell-github
       - name: prettier
         run: yarn lint:prettier
+  fossa-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)